vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -245,6 +245,11 @@ def invoke_fused_moe_kernel(A: torch.Tensor, B: torch.Tensor, C: torch.Tensor,
     )
 
 
+def get_json_file_name(E: int, N: int) -> str:
```

Comment:
nit?
```suggestion
def get_config_file_name(E: int, N: int) -> str:
```

Reply:
Fixed in [005eaf2](https://github.com/vllm-project/vllm/pull/3389/commits/005eaf2ab7ec0c4c15fdfaeb1852149315c84e0f).